src/posix/platform/infra_if.cpp

```diff
@@ -296,20 +296,33 @@ void InfraNetif::Init(otInstance *aInstance, const char *aIfName)
 
     mNetLinkSocket = CreateNetLinkSocket();
 
-    SuccessOrDie(otBorderRoutingInit(aInstance, ifIndex, platformInfraIfIsRunning()));
-    SuccessOrDie(otBorderRoutingSetEnabled(aInstance, /* aEnabled */ true));
+exit:
+    return;
+}
 
-    mInstance = aInstance;
-    Mainloop::Manager::Get().Add(*this);
+void InfraNetif::SetUp(void)
+{
+    OT_ASSERT(gInstance != nullptr);
+    VerifyOrExit(mInfraIfIndex != 0);
```

Comment:
Here and everywhere, shall we prefer to use `otSysGetInstance()`?

Reply:
What do you mean by using `otSysGetInstance`?

BTW, otSysGetInstance is now removed.

Reply:
We don't use `otSysGetInstance` in posix platform code because `gInstance` is available.